src/test/java/com/example/UserServiceTest.java

```diff
@@ -0,0 +1,22 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I would recommend optimising it by only importing the Assertions needed within this PR.